### ISSUE TITLE
Error following authentication

### ISSUE THREAD

**Issue by MikeHarrison** [genxbe/kirby3-instagram#5]
Hi,
I think everything is working correctly with the plugin, I have done a test fetch which worked. However, when the authentication triggered in Kirby's panel was completed I got the following error:

```
Whoops\Exception\ErrorException thrown with message "Cms\Site::panelUrl() has been deprecated and will be removed in Kirby 3.8.0. Use $site->panel()->url() instead."

Stacktrace:
#9 Whoops\Exception\ErrorException in /home/***/dev/kirby/src/Cms/Helpers.php:29
#8 trigger_error in /home/***/dev/kirby/src/Cms/Helpers.php:29
#7 Kirby\Cms\Helpers:deprecated in /home/***/dev/kirby/src/Cms/Site.php:696
#6 Kirby\Cms\Site:panelUrl in /home/***/dev/site/plugins/kirby3-instagram/classes/Instagram.php:86
#5 X\Instagram:getToken in /home/***/dev/site/plugins/kirby3-instagram/index.php:21
#4 Kirby\Http\Route:{closure} in [internal]:0
#3 Closure:call in /home/***/dev/kirby/src/Http/Router.php:131
#2 Kirby\Http\Router:call in /home/***/dev/kirby/src/Cms/App.php:337
#1 Kirby\Cms\App:call in /home/***/dev/kirby/src/Cms/App.php:1256
#0 Kirby\Cms\App:render in /home/***/dev/index.php:5
```

I am running Kirby 3.7.1

**Comment by samzzi** [comment]
Hi @MikeHarrison apologies for the late reply, I was on vacation ;)
panelUrl is indeed deprecated in 3.7.1. I'll update the code this week and push the fix.

**Comment by samzzi** [comment]
Fixed in 1.0.7

**Comment by MikeHarrison** [comment]
Super, thanks!
